src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -27,6 +37,7 @@ import { calcStartPosition } from "components/molecules/Schedule/utils";
 import "./ScheduleEvent.scss";
 
 const ScheduleEventBookmarkClass = "ScheduleEvent__bookmark";
+const ScheduleEventExpandClass = "ScheduleEvent__expand";
```

Comment:
I suppose we don't need to store it anymore (as this const was created for `e.target.closest()` ).

Reply:
removed 64c0c27